r124/u_inref/ydb312_gtm8182c.csh

```diff
@@ -0,0 +1,119 @@
+#!/usr/local/bin/tcsh -f
+#################################################################
+#								#
+# Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	#
+# All rights reserved.						#
+#								#
+#	This source code contains the intellectual property	#
+#	of its copyright holder(s), and is made available	#
+#	under a license.  If you do not know the terms of	#
+#	the license, please stop and do not read further.	#
+#								#
+#################################################################
+#
+setenv gtm_repl_instance "mumps.repl"
+
+# Assign the KEY2BIG error to the list of 'serious' errors worth freezing the DB for
+cp $ydb_dist/custom_errors_sample.txt .
+setenv gtm_custom_errors `pwd`/custom_errors_sample.txt
+chmod +w `pwd`/custom_errors_sample.txt
+echo "KEY2BIG" >> `pwd`/custom_errors_sample.txt
+unsetenv ydb_custom_errors
+
+# once the error is generated we'll freeze the associated instances DB
+setenv gtm_test_freeze_on_error 1
+# global key of >5 characters will trigger the KEY2BIG error
+setenv keysize 5
```

Comment:
I would prefer using -key_size=5 in the dbcreate.csh command line instead of just 5. Former is more readable code.